compiler/suggest.nim

```diff
@@ -11,61 +11,88 @@
 
 # included from sigmatch.nim
 
-import algorithm, sequtils
+import algorithm, sequtils, strutils, ast, msgs, options, renderer,
+  types, docgen, lexer, semdata, astalgo, idents, sigmatch, lookups
```

Comment:
If suggest.nim is included from sigmatch, why do you import sigmatch here? And in fact, I'm sure these deps shouldn't be here.